pubsub/pubsub.go

```diff
@@ -40,3 +54,26 @@ func Ping(pubsub PubSub) error {
 		return fmt.Errorf("ping is not implemented by this pubsub")
 	}
 }
+
+// DefaultBulkMessager is default implemnetation for BukMessager
+type DefaultBulkMessager struct {
+	p PubSub
+}
+
+// NewDefaultBulkMessager to create new DefaultBulkMessager for a PubSub
+func NewDefaultBulkMessager(pubsub PubSub) DefaultBulkMessager {
+	return DefaultBulkMessager{
+		p: pubsub,
+	}
+}
+
+// BulkPublish Default Implementation
+func (p *DefaultBulkMessager) BulkPublish(req *BulkPublishRequest) (BulkPublishResponse, error) {
+	return BulkPublishResponse{}, nil
+}
+
+// BulkSubscribe Default Implementation
+func (p *DefaultBulkMessager) BulkSubscribe(tx context.Context, req SubscribeRequest,
+	handler BulkHandler) error {
+	return nil
```

Comment:
should this return an unimplemented error for now in this PR? And have a follow up PR to complete the implementation?

Reply:
I am okay with either.

Reply:
I too am - can change error conveying unimplemented